### ISSUE TITLE
HologramClickEvent not called when clicking a hologram

### ISSUE THREAD

**Issue by Luncaaa** [DecentSoftware-eu/DecentHolograms#122]
### Just making sure

- [X] I made sure, this bug is not already known or reported.
- [X] I believe, that this bug is fixable.

### Reproduction

1. Create a plugin with DecentHolograms as a dependency
2. Create a hologram
3. Create a listener for the HologramClickEvent 
4. Click the hologram created in step 3 and nothing will happen (no errors and the event is not called)

This is a simple event listener I created which does not work:
Listener class
```java
public class Test implements Listener {
    @EventHandler
    public void onHologramClick(HologramClickEvent event) {
        Bukkit.getLogger().info("Hologram clicked");
    }
}
```
Main class:
```java
getServer().getPluginManager().registerEvents(new Test(), this);
```

### Server Version

Paper 1.19.3 #446

### Client Version

1.19.3

### Plugin Version

2.7.12

### Log

https://pastebin.com/LVK4gqK1

**Comment by Andre601** [comment]
My best guess here why the click event no longer works, is that the fix for removing the hitbox, which caused you to not be able to interact with blocks/NPCs below the hologram, makes it impossible to click a hologram, because there's no longer a hitbox to click...

If that's the case, then we have an interesting issue here where you either re-add the hitboxes, making the old issues with NPCs and blocks appear again, or not re-add them, causing the issues you encounter. :/

**Comment by Luncaaa** [comment]
Would it be possible to add an option in the config file which allows you to enable or disable the hitboxes?

**Comment by d0by1** [comment]
Hello, this has been fixed in version 2.8.1. Thanks for reporting it!
